controllers/user.js

```diff
@@ -43,8 +44,37 @@ const deleteUser = async (req, res) => {
   }
 };
 
+const updateUserByID = async (req, res) => {
+  const id = req.params.id;
```

Comment:
We should probably use the one on res.locals (And use the verify-token middleware for that route), because only the same user can update himself

Reply:
Done, any controller using the ID as a parameter was updated accordingly